Gemfile

```diff
@@ -2,6 +2,8 @@
 
 source 'https://rubygems.org'
 
+gem 'aws-sdk-core'
```

Comment:
This isn't related to hiera upgrade, please move to a separate feature branch / PR.